src/AWSTools.jl

```diff
@@ -1,15 +1,16 @@
 module AWSTools
 
-using AWSCore
-using AWSCore.Services: sts
+using AWS
 using AWSS3
 using Mocking
 using Random
 using Dates: Period, unix2datetime
 export assume_role
 
-get_caller_identity() = sts("GetCallerIdentity")
-account_id() = (@mock get_caller_identity())["Account"]
+@service STS
+
+get_caller_identity() = STS.get_caller_identity()
```

Comment:
```suggestion
get_caller_identity() = STS.get_caller_identity()["GetCallerIdentityResult"]
```

IMO this will help a lot with transitioning and I can't see why people would want the outer key anyway.

Reply:
Yeah... this outer layer is really silly. But only *some* services do this 🙄